Onboarding/re-engagement.md

```diff
@@ -0,0 +1,431 @@
+<!-- { section: "97ca43be-ba28-49df-98f1-4207014d0d65", x: 500, y: 48} -->
+
+```stack
+trigger(on: "MESSAGE RECEIVED") when has_only_phrase(event.message.text.body, "engage")
+
+```
+
+```stack
+# card Setup, then: FetchError do
+#   update_contact(reengagement_message: "1st message")
+# end
+
+```
+
+<!-- { section: "9a4ecd16-1072-4929-8f8f-5f1c63cdd452", x: 0, y: 0} -->
+
+```stack
+card FetchError, then: DropOffRedirect do
+  # Fetch and store the error message, so that we don't need to do it for every error card
+  search =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/",
+      query: [
+        ["slug", "mnch_onboarding_error_handling_button"]
+      ],
+      headers: [["Authorization", "Token @global.config.contentrepo_token"]]
+    )
+
+  # We get the page ID and construct the URL, instead of using the `detail_url` directly, because we need the URL parameter for `get` to start with `https://`, otherwise stacks gives us an error
+  page_id = search.body.results[0].id
+
+  page =
+    get(
+      "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/pages/@page_id/",
+      query: [
+        ["whatsapp", "true"]
+      ],
+      headers: [["Authorization", "Token @global.config.contentrepo_token"]]
+    )
+
+  button_error_text = page.body.body.text.value.message
+end
+
+```
+
+## Drop Off Redirect And Schedule Reminder
+
+```stack
+card DropOffRedirect when contact.reengagement_message == "1st message", then: DropOff2ndReminder do
+  text("To send 2nd message")
```

Comment:
Probably need to remove this?

Reply:
Hi @HawkiesZA why do we need to remove these checks as they are for redirecting which message we need to send to a user?